src/main/java/uk/gov/companieshouse/search/api/service/search/impl/alphabetical/AlphabeticalSearchRequestService.java

```diff
@@ -177,21 +186,20 @@ private Company getCompany(SearchHit hit) {
         Map<String, Object> links = (Map<String, Object>) sourceAsMap.get("links");
 
         Company company = new Company();
-        Items companyItems = new Items();
         Links companyLinks = new Links();
 
-        companyItems.setCorporateName((String) (items.get("corporate_name")));
-        companyItems.setCompanyNumber((String) (items.get("company_number")));
-        companyItems.setCompanyStatus((String) (items.get("company_status")));
-        companyItems.setOrderedAlphaKey((String) items.get("ordered_alpha_key"));
-        companyItems.setOrderedAlphaKeyWithId((String) sourceAsMap.get(ORDERED_ALPHA_KEY_WITH_ID));
+        company.setCompanyName((String) (items.get("corporate_name")));
+        company.setCompanyNumber((String) (items.get("company_number")));
+        company.setCompanyStatus((String) (items.get("company_status")));
+        company.setOrderedAlphaKey((String) items.get("ordered_alpha_key"));
+        company.setOrderedAlphaKeyWithId((String) sourceAsMap.get("ordered_alpha_key_with_id"));
+        company.setKind((String) sourceAsMap.get("kind"));
 
         companyLinks.setSelf((String) (links.get("self")));
+        company.setLinks(companyLinks);
 
-        company.setId((String) sourceAsMap.get("ID"));
+        // company.setId((String) sourceAsMap.get("ID"));
```

Comment:
Can this commented out line be removed or is it still required?

Reply:
I removed this as part of the sonar issues commit?